pkgs/development/python-modules/pyliblo/default.nix

```
@@ -23,6 +23,11 @@ buildPythonPackage rec {
       url = "https://git.alpinelinux.org/aports/plain/community/py3-pyliblo/py3.11.patch?id=a7e1eca5533657ddd7e37c43e67e8126e3447258";
       hash = "sha256-4yCWNQaE/9FHGTVuvNEimBNuViWZ9aSJMcpTOP0fnM0=";
     })
+    # Fix compile error due to  incompatible pointer type 'lo_blob_dataptr'
+    (fetchurl {
+      url = "https://github.com/dsacre/pyliblo/commit/ebbb255d6a73384ec2560047eab236660d4589db.patch?full_index=1";
```

GitHub warns that this commit is not in the main repository. We should probably put the fork here to be more explicit of the patch provenance.

```suggestion
      url = "https://github.com/s0600204/pyliblo/commit/ebbb255d6a73384ec2560047eab236660d4589db.patch?full_index=1";
```